crates/cli_testing_examples/benchmarks/CFold.roc

```diff
@@ -100,35 +100,27 @@ constFolding = \e ->
             x1 = constFolding e1
             x2 = constFolding e2
 
-            when Pair x1 x2 is
-                Pair (Val a) (Val b) ->
-                    Val (a + b)
+            when x1 is
+                Val a ->
+                    when x2 is
+                        Val b -> Val (a + b)
```

Comment:
Do these changes make the program more amenable to TRMC, or is this for another reason?

Reply:
I briefly thought so, but no. This is more in line with koka, so it is easier to compare our output to theirs.